samples/CdnSignedUrlSignatureComputationSample/SignedUrlSignature.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Text;
+using System.Security.Cryptography;
+
+namespace CdnSignedUrlSignatureComputationSample
+{
+    class SignedUrlSignature
+    {
+        /// <summary>
+        /// Converts input to URL safe base64 string.
+        /// </summary>
+        /// <param name="signature">Computed signature.</param>
+        /// <returns>Url safe base64 signature.</returns>
+        private string ConvertToUrlSafeBase64String(byte[] signature)
+        {
+            return Convert.ToBase64String(signature).Replace('/', '_').Replace('+', '-');
+        }
+
+        /// <summary>
+        /// Function to compute signature.
+        /// </summary>
+        /// <param name="resourcePath">Resource Path.</param>
```

Comment:
It would be helpful to flesh out the doc comments with additional context.   For example, what is an `Expires parameter value`?  What effect does it have on the generated value?

Reply:
Added more doc comments.